build_matrix.yml

```diff
@@ -111,7 +111,7 @@ semaphore: # Default `.semaphore/semaphore.yml` contents
         - name: Diagnose
           env_vars:
             - name: RUBY_VERSION
-              value: 2.6.6
+              value: 3.0.2
```

Comment:
Could we drop this environment variable entirely, if Semaphore automatically chooses the right Ruby based on the `.ruby-version`?

Reply:
Currently these env vars are used to generate the cache keys. Not sure if you get any conflicts with other jobs if this env var is not set.